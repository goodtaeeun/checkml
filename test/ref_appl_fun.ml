List.iter
  (fun x ->
    let a = ref 43 in
    ())
  []
