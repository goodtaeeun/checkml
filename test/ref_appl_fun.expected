File ref_appl_fun.ml, line 3, column 12
[31mError[0m: Don't use ref!

